kratos/python/add_registry_to_python.h

```diff
@@ -0,0 +1,29 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//                   Carlos Roig
+//                   Ruben Zorrilla
+//
+
+#pragma once
+
+// System includes
+#include <pybind11/pybind11.h>
```

Comment:
```suggestion
#include "includes/define_python.h"
```

Reply:
Why we need the full define_python.h here? We are not using custom holders or register functions

Reply:
good point
I guess I would not include the ext lib stuff diretly, but it is used in this file